site/resources.md

```diff
@@ -6,33 +6,63 @@ id: resources
+
+<iframe title="How to Backup and Restore Your Kubernetes Cluster - Annette Clewett & Dylan Murray, Red Hat" width="560" height="315" src="https://www.youtube.com/embed/JyzgS-KKuoo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</br>
+* Kubernetes Back Up, Restore and Migration with Velero - A NewStack interview, with guests: Carlisia Campos, Tom Spoonemore, and Efri Nattel-Shayo:
+
+<iframe title="Kubernetes Back Up, Restore and Migration with Velero - NewStack, with guests: Carlisia Campos, Tom Spoonemore, and Efri Nattel-Shayo" width="560" height="315" src="https://www.youtube.com/embed/71NoY5CIcQ8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</br>
+* How to migrate applications between Kubernetes clusters using Velero:
+
+<iframe title="How to migrate applications between Kubernetes clusters using Velero" width="560" height="315" src="https://www.youtube.com/embed/IZlwKMoqBqE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</br>
+* TGIK 080: Velero 1.0:
+
+<iframe title="TGIK 080: Velero 1.0" width="560" height="315" src="https://www.youtube.com/embed/tj5Ey2bHsfM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</br>
 * Watch our recent webinar on backup and migration strategies:
 
 <iframe title="Kubernetes Backup and Migration Strategies using Project Velero" width="560" height="315" src="https://www.youtube.com/embed/csrSPt3HFtg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</br>
+* Velero demo by Just me and Opensource - [ Kube 45 ] Velero - Backup & Restore Kubernetes Cluster:
+
+<iframe title="[ Kube 45 ] Velero - Backup & Restore Kubernetes Cluster" width="560" height="315" src="https://www.youtube.com/embed/C9hzrexaIDA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 
 ## Blog posts
 
+* How to Make Up for Kubernetes’ Disaster Recovery Shortfalls - by Carlisia Campos
+    * https://thenewstack.io/how-to-make-up-for-kubernetes-disaster-recovery-shortfalls/
+</br>
+* Project Velero 1.2 on OpenShift 3.11 - Backup and Restore (and backup to NFS) - by Raael Brito
+    * http://www.rafaelbrito.com/2019/11/project-velero-12-on-openshift-311.html
+</br>
+* Backing Up Your Kubernetes Applications with Velero v1.1 - by Joe Mann
+    * https://mannimal.blog/2019/10/04/backing-up-your-kubernetes-applications-with-velero-v1-1/
+</br>
+* Using Velero for K8s Backup and Restore of CSI Volumes - by Myles Gray
+    * https://blah.cloud/automation/using-velero-for-k8s-backup-and-restore-of-csi-volumes/
+</br>
```

Comment:
You can remove the `</br>` here, and just use two spaces/one tab for the sub-bullets